internal/variables/variables.go

```diff
@@ -222,12 +222,20 @@ const (
 	MultipartUnmatchedBoundary
 	// PathInfo is kept for compatibility
 	PathInfo
-	// Sessionid is not supported
+	// Sessionid is the session id
 	Sessionid
-	// Userid is not supported
+	// Userid is a persistent collection of user ids
```

Comment:
It's simply wrong. Could you add comments based on [the official documentation](https://github.com/owasp-modsecurity/ModSecurity/wiki/Reference-Manual-%28v3.x%29#user-content-USERID)?
![image](https://github.com/user-attachments/assets/7ff35971-c253-4a13-9ff2-75a872abf6dd)